models/attachment/src/index.ts

```diff
@@ -80,8 +82,19 @@ export class TSavedAttachments extends TPreference implements SavedAttachments {
   declare attachedTo: Ref<Attachment>
 }
 
+@Model(attachment.class.Drawing, core.class.Doc, DOMAIN_ATTACHMENT)
+export class TDrawing extends TDoc implements Drawing {
+  @Prop(TypeRef(core.class.Doc), getEmbeddedLabel('Parent'))
+  @Index(IndexKind.Indexed)
+  @Hidden()
+    parent!: Ref<Doc>
+
+  @Prop(TypeString(), getEmbeddedLabel('Content'))
```

Comment:
If content is file please use Ref<Blob> instead

Reply:
I think we decided on not to use files for drawings because they will only clutter the space. So this is not a reference to a file, but serialized drawing content. An object the drawing is attached to is in the `parent` field

Reply:
ok.